eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -127,7 +130,17 @@ jobs:
             TestType: "node"
             DependencyVersion: same
             NodeTestVersion: "12.x"
-            TestResultsFiles: "**/test-results.xml"            
+            TestResultsFiles: "**/test-results.xml"   
+
+        # Add matrix entry for canary testing
+        ${{ if eq(parameters.TestCanary, 'true') }}:
+          CanaryTest_Node:
+            OSVmImage: "ubuntu-18.04"
+            TestType: "node"
+            NodeTestVersion: "12.x"
+            TestResultsFiles: "**/test-results.xml"
+            SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+            ResourceGroupLocation: centraluseuap
```

Comment:
Ideally I would like to see this as a config param so we can override it if required. Currently there is no way to do if we hardcode it here. Can you please make this as a param like TestCanary and set "centraluseuap" as default value